pg-index-health/src/test/java/io/github/mfvanek/pg/support/statements/CreateSchemaStatement.java

```diff
@@ -13,16 +13,18 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.util.List;
 import javax.annotation.Nonnull;
 
 public class CreateSchemaStatement extends AbstractDbStatement {
 
-    public CreateSchemaStatement(@Nonnull final String schemaName) {
-        super(schemaName);
+    @Override
+    protected List<String> getSqlToExecute() {
+        return List.of("create schema if not exists " + getSchemaName());
     }
 
     @Override
-    public void execute(@Nonnull final Statement statement) throws SQLException {
+    public void postExecute(@Nonnull final Statement statement, @Nonnull final String schemaName) throws SQLException {
         statement.execute("create schema if not exists " + schemaName);
```

Comment:
Remove line `statement.execute("create schema if not exists " + schemaName);` from `postExecute`